docs/api-reference/data-fetching/get-static-paths.md

```diff
@@ -52,6 +54,7 @@ The value for each `params` object must match the parameters used in the page na
 - If the page name is `pages/posts/[postId]/[commentId]`, then `params` should contain `postId` and `commentId`.
 - If the page name uses [catch-all routes](/docs/routing/dynamic-routes.md#catch-all-routes) like `pages/[...slug]`, then `params` should contain `slug` (which is an array). If this array is `['hello', 'world']`, then Next.js will statically generate the page at `/hello/world`.
 - If the page uses an [optional catch-all route](/docs/routing/dynamic-routes.md#optional-catch-all-routes), use `null`, `[]`, `undefined` or `false` to render the root-most route. For example, if you supply `slug: false` for `pages/[[...slug]]`, Next.js will statically generate the page `/`.
+- `locale`: You can also use the `locale` string if you intend to change the active locale.
```

Comment:
Should we move this to the `getStaticPaths` API docs instead? We previously had a lot of this here, but have since extracted it to that section 😄

Reply:
By the way, thanks for adding this following the discussion in https://github.com/vercel/next.js/issues/32772 but this was not even what I meant by missing documentation based on @balazsorban44's [reply](https://github.com/vercel/next.js/issues/32772#issuecomment-1013153332).  He added the path before the `param` which I am not even sure today what it does.

I know the issue has been closed but would recommend taking a second look because I am still seeing a weird behavior with `asPath`!

Reply:
>  I am not even sure today what it does.

@nbouvrette, @MaedahBatool This lets you render pages with a given locale at build-time.

> recommend taking a second

Sure, I'll try to have another look, thanks.

Reply:
Makes sense @balazsorban44. Please let me know when you've reviewed it. I'll then fix it accordingly. :)

Reply:
@balazsorban44 Thanks for taking a second look.

By the way to make sure we are talking about the same thing - I was not talking about the locale parameter, I was talking about this one you added:

![image](https://user-images.githubusercontent.com/6453322/149967330-341d37d6-24ec-4f14-992e-dfe55f533419.png)